plugin/adsk/plugin/geomNode.h

```diff
@@ -0,0 +1,73 @@
+//
+// Copyright 2022 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#ifndef PXRUSDMAYA_MAYAUSDGEOMNODE_H
+#define PXRUSDMAYA_MAYAUSDGEOMNODE_H
+
+#include "base/api.h"
+
+#include <mayaUsd/base/api.h>
+
+#include <maya/MObject.h>
+#include <maya/MPxNode.h>
+#include <maya/MStatus.h>
+
+#include <memory>
+#include <vector>
+
+PXR_NAMESPACE_USING_DIRECTIVE
+
+namespace MAYAUSD_NS_DEF {
+class MayaUsdGeomNode : public MPxNode
```

Comment:
A comment for the purpose of the class. I know we're not always examplar in doing this ourselves... :)